nielsendtvr/build.gradle

```diff
@@ -0,0 +1,40 @@
+apply plugin: 'com.android.library'
+
+android {
+    compileSdkVersion 28
+
+    defaultConfig {
+        minSdkVersion 14
+        targetSdkVersion 28
+        versionCode 1
+        versionName "1.0"
+    }
+
+    compileOptions {
+        sourceCompatibility JavaVersion.VERSION_1_8
+        targetCompatibility JavaVersion.VERSION_1_8
+    }
+
+    testOptions {
+        unitTests {
+            all {
+                jvmArgs '-noverify'
+            }
+            includeAndroidResources = true
+        }
+    }
+}
+
+dependencies {
+    repositories {
+        google()
+        jcenter()
+    }
+
+    implementation files('appsdk/appsdk.jar')
```

Comment:
Would be possible to add the jar to the repo?

Reply:
I'm not sure on the whole legality/licensing behind sharing the Nielsen sdk publicly. Normally you do need to fill out a form first to download it. But this is something that I assume would need confirmation from Nielsen.